index.ts

```diff
@@ -15,7 +15,7 @@ import ObserveWriteStream from './lib/observe_write_stream'
 import { parameters, refreshTiming, defaultTiming } from './lib/parameters'
 import { isIPv6 } from 'net'
 import { registerOption, registerFormat, ignoreOption } from './lib/option_converter'
-import { CoapServerOptions, requestListener, CoapRequestParams, ParametersUpdate, AgentOptions, CoapPacket, Option, OptionValue } from './models/models'
+import { type CoapServerOptions, type requestListener, type CoapRequestParams, type ParametersUpdate, type AgentOptions, type CoapPacket, type Option, type OptionValue } from './models/models'
```

Comment:
Would not "import type {...}" also work and be more clear?

Reply:
Thank you :) I wasn't aware of this possibility, should now be fixed.